windows/bleachbit.nsi

```diff
@@ -56,6 +58,44 @@
 !endif
 
 
+;--------------------------------
+;MultiUser defines
+
+; https://github.com/Drizin/NsisMultiUser/wiki/Defines
+!define PRODUCT_NAME "${prodname}" ; exact copy to another name for multi-user script
+; !define VERSION "2.3"
+; "VERSION" already defined!
+!define PROGEXE "${prodname}.exe"
+; !define COMPANY_NAME "BleachBit"
+; "COMPANY_NAME" already defined!
+
+; An option (MULTIUSER_INSTALLMODE_ALLOW_BOTH_INSTALLATIONS) defines whether simultaneous per-user
+; and per-machine installations on the same machine are allowed. If set to disallow, the installer
+; alaways requires elevation when there's per-machine installation in order to remove it first.
+!define MULTIUSER_INSTALLMODE_ALLOW_BOTH_INSTALLATIONS 1
```

Comment:
Please set ALLOW BOTH to 0 to reduce confusion and complexity. It seems extremely rare that anyone would want two installations. Even when I do testing, I don't have two installations. If I want to test two versions, I might use portable versions.

Reply:
But as far as I understand the description, in this case you can't install system wide when a user have BB already installed.
I thought e.g. in a corporate environment the admin comes always first and the user that have installed BB before must later deinstall second version... Was just my view as an old admin...

Reply:
@Tobias-B-Besemer 
In a corporate environment, I trust that the admin would keep BleachBit up to date. If not, the user could use a portable version.  However, generally users shouldn't try to change what admins do. The admin might not appreciate it. Any any case, I think this situation would be rare.